src/main/java/io/github/thebusybiscuit/slimefun4/core/services/sounds/SoundConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package io.github.thebusybiscuit.slimefun4.core.services.sounds;
+
+import javax.annotation.Nonnull;
+
+/**
+ * This structure class holds configured values for a {@link SoundEffect}.
+ * 
+ * @author TheBusyBiscuit
+ * 
+ * @see SoundService
+ * @see SoundEffect
+ *
```

Comment:
there are empty lines in the end of javadocs all around here, is there a reason?

Reply:
Resolved in #3694